admin-dev/themes/default/scss/partials/_product.scss

```diff
@@ -1,19 +1,20 @@
+// @TODO still used ?
```

Comment:
This was probably used in the old page, the product page has been migrated (twice even ^^) So I think this could be cleaned, and probably `js/admin/products.js` as well

Reply:
@jolelievre Daniel started working on this: https://github.com/PrestaShop/PrestaShop/pull/36110

but as I pointed out, it's pretty tricky because some of the features are widely used in modules from the ecosystem

anyway, this is one is safe to remove in my opinion, but still we should keep track of such changes and make sure they go to the devdocs 👍🏻